juntagrico/lifecycle/member.py

```diff
@@ -0,0 +1,31 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext as _
+
+from juntagrico.config import Config
+from juntagrico.signals import member_deactivated
+
+
+def member_pre_save(sender, instance, **kwargs):
+    if instance._old['inactive'] != instance.inactive and instance._old['inactive'] is False:
```

Comment:
Probably `instance.inactive is True` would be more intuitive to read. Also elsewere.